### PR TITLE
Add Haxe JVM

### PR DESCRIPTION
**Pull request by barisyild**
This haxe target translates the source language into JVM bytecode like Kotlin.

### CODE REVIEW

**Comment by dimensionscape** [comment]
Nice! Haxe doesn't get enough visibility. Totally underrated!

File: loops/haxe/Code.hx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class Code {
+	public static function main() {
+		var u:Int = Std.parseInt(Sys.args()[0]); // Command line input number
+		var r:Int = Std.random(10000);
+		var a = new Vector<Int>(10000); // Initialize a vector with 10,000 elements
+
+		for (i in 0...10000) { // Outer loop with 10k iterations
+			#if !java
+			a[i] = 0; // Set all VectorData to 0, this part is unnecessary since the JVM target already fills the values with 0.
```
Comment (bddicken) [inline loops/haxe/Code.hx:11]: The zeroing out needs to be done before the loop to match the other test cases, so this should be removed / changed.

Comment (barisyild) [reply loops/haxe/Code.hx]: https://github.com/bddicken/languages/pull/70/commits/0b76083104c507040c01c80e17e30cf570b3dd4d

Fixed

**Comment by bddicken** [comment]
Cool, thanks!

**Pull request merged**